Gruntfile.js

```diff
@@ -350,16 +346,18 @@ module.exports = function(grunt) {
 			}
 		},
 		run: {
-			npm_run_imports: {
-				cmd: 'node',
-				args: ['./build/imports-generator']
+			npmRunImportsGenerator: {
+				exec: 'npm run imports-gen'
+			},
+			npmRunTestHeadless: {
```

Comment:
Please remove this. We do not need Grunt to run npm scripts.

Reply:
We do until the day we lose `Grunt` and move solely to `npm scripts`, in order to maintain the sequence in which tasks are executed. For example the test execution in headless mode depends on `build`, `testconfig`, `fixture`, `connect`, which are steps for building `axe`, generating test fixtures based on runner templates and booting a server respectively.

Reply:
@jkodu you can put those tasks in an NPM script. I agree with Stephen. We should transition out of Grunt.